internal/scorecard/plugins/olm_tests.go

```diff
@@ -87,19 +111,19 @@ func NewCRDsHaveResourcesTest(conf OLMTestConfig) *CRDsHaveResourcesTest {
 	}
 }
 
-// AnnotationsContainExamplesTest is a scorecard test that verifies that the CSV contains examples via the alm-examples annotation
-type AnnotationsContainExamplesTest struct {
+// CSVValidationTest is a scorecard test that validates a CSV
+type CSVValidationTest struct {
 	schelpers.TestInfo
 	OLMTestConfig
 }
 
-// NewAnnotationsContainExamplesTest returns a new AnnotationsContainExamplesTest object
-func NewAnnotationsContainExamplesTest(conf OLMTestConfig) *AnnotationsContainExamplesTest {
-	return &AnnotationsContainExamplesTest{
+// CSVValidationTest returns a new ValidationTest object
+func NewCSVValidationTest(conf OLMTestConfig) *CSVValidationTest {
+	return &CSVValidationTest{
 		OLMTestConfig: conf,
 		TestInfo: schelpers.TestInfo{
-			Name:        "CRs have at least 1 example",
-			Description: "The CSV's metadata contains an alm-examples section",
+			Name:        "CSV validates",
+			Description: "Validate the CSV",
```

Comment:
Should not be better to keep a description more detailed about what actually is tested/checked at this point?